docker.nix

```diff
@@ -125,6 +125,7 @@ let
     sandbox = "false";
     build-users-group = "nixbld";
     trusted-public-keys = "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=";
+    ignored-acls = "security.csm";
```

Comment:
Hm, as @cole-h said, we should remove this since it's not needed. In fact it has the effect of overriding the other built-in ignored ACLs (which could be avoided by using `extra-ignored-acls`).
```suggestion
```